doc/source/parameters/dem/model_parameters.rst

```diff
@@ -0,0 +1,92 @@
+Model Parameters
+-------------------
+In this subsection, DEM simulation parameters are defined. These parameters include ``contact detection method`` and its subsequent information (``dynamic contact search size coefficient`` **or** ``contact detection frequency`` for ``dynamic`` **or** ``constant`` contact detection method), ``repartition method`` (``once``, ``frequent`` or ``dynamic``) for ``load balancing`` (in parallel simulations), ``particle weight`` which defines the particle weight compared to a default ``cell weight`` of 1000, ``neighborhood threshold`` (which defines the contact neighbor list size: ``neighborhood threshold`` * particle diameter), ``particle particle contact force method``, ``particle wall contact force method`` and ``integration method``. 
+
+.. note::
+    In ``constant`` contact search, Lethe-DEM searches for particle-particle and particle-wall collisions at a constant frequency (every ``contact detection frequency`` iterations). Normally, ``contact detection frequency`` should be selected in the range of 5-50. Small values of ``contact detection frequency`` lead to long simulation times, while large values of ``contact detection frequency`` may lead to late detection of collisions. Late detection of collisions can result in very large particles velocities (popcorn jump of particles in a simulation) or particles leaving the simulation domain.
+
+.. note::
+    In ``dynamic`` contact search, Lethe-DEM searches for particle-particle and particle-wall collisions automatically. In ``dynamic`` contact search, Lethe-DEM predicts (using the velocities of particles at each iteration) and stores (adds up) the displacements of each particle in the simulation. If the maximum displacement of particles exceeds the smallest contact search criterion (explained in the following), then the iteration is recognized as a contact detection iteration.
```

Comment:
I would try something like this : 

```suggestion
Particle-particle contact search is a costly operation in DEM simulations. If the contact detection is not carried out as optimally as possible, DEM simulations can be slowed down significantly. Lethe supports two mechanism to detect the contact between particles. 
* By setting ``contact detection method = constant``. contact search will be carried out at constant frequency (every ``contact detection frequency`` iterations). Normally, the ``contact detection frequency`` should be a value between 5 and 50. The contact frequency should be chosen such that the particles do not travel more than half a cell between two contact detection. Small values of ``contact detection frequency`` lead to long simulation times, while large values of ``contact detection frequency`` may lead to late detection of collisions. Late detection of collisions can result in very large particles velocities (popcorn jump of particles in a simulation) or particles leaving the simulation domain.
* By setting ``contact detection method = dynamic``, Lethe-DEM rebuilds the contact lists automatically. In this mode, Lethe-DEM stores the displacements of each particle in the simulation since the last contact detection. If the maximum displacement of a particle exceeds the smallest contact search criterion (explained in the following), then the iteration is a contact search iteration and the contact list is rebuilt.
```